docs/proposals/variable_changes.md

```diff
@@ -0,0 +1,308 @@
+# Current Situation
+
+First, openshift.common.use_calico is set.  This is created from the inventory
```

Comment:
I would actually add an abstract introduction above this. It seems based on this what you're proposing is to move away from `openshift_facts` because we have variables that are 1:1 with it (the example being `openshift.common.use_calico` and `openshift_use_calico`) among other reasons we should get rid of it (that is a long list that pretty much everyone agrees on at this point)

Reply:
It also seems like you're proposing that we work to be more consistent with variable usage and how its passed on since there are multiple places a value may be set and so it may have a different value at different points in a playbook execution

Reply:
Yeah, I was mentioning that lets just assume that openshift_facts goes away for the most part. I think it'll be easier to discuss if we limit the scope here.